### PR TITLE
Add additional sketch that use v_percentage

### PR DESCRIPTION
**Pull request by gryzli133**
I start developing this code some years ago. Now I use it since 3 years and it works well. I have over 30 covers running on 3 Arduino Mega boards. The biggest advantage is, that the local buttons works always, independent of the gateway or smart home server state/condition.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @gryzli133, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21121).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/cover.mysensors.markdown [hunk 1]
```diff
@@ -149,3 +149,9 @@ void receive(const MyMessage &message) {
   }
 }
 ```
+
+## Sketch example with position measurement based on motor running time
+
+This sketch is ideally for star topology wiring. You can run up to 12 covers with a single Arduino Mega board and some releys. All you need to set is one line of parameters for one Cover. However you can also use it for a single cover based on an arduino nano or even ESP8266 board.
+
+https://github.com/gryzli133/RollerShutterSplit
```
Comment (frenck) [inline source/_integrations/cover.mysensors.markdown]: This is a bare link, please don't use those.

Comment (gryzli133) [reply source/_integrations/cover.mysensors.markdown]: the sketch is built of multiple files, so I cannot easily paste the code...

Comment (frenck) [reply source/_integrations/cover.mysensors.markdown]: That is fine, but please don't use bare links.

Comment (gryzli133) [reply source/_integrations/cover.mysensors.markdown]: I changed that. Should be OK now.

File: source/_integrations/cover.mysensors.markdown [hunk 2]
```diff
@@ -149,3 +149,9 @@ void receive(const MyMessage &message) {
   }
 }
 ```
+
+## Sketch example with position measurement based on motor running time
+
+This sketch is ideally for star topology wiring. You can run up to 12 covers with a single Arduino Mega board and some releys. All you need to set is one line of parameters for one Cover. However you can also use it for a single cover based on an arduino nano or even ESP8266 board.
```
Comment (frenck) [inline source/_integrations/cover.mysensors.markdown]:
```suggestion
This sketch is ideally for star topology wiring. You can run up to 12 covers with a single Arduino Mega board and some relays. All you need to set is one line of parameters for one Cover. However, you can also use it for a single cover based on an Arduino Nano or even an ESP8266 board.
```

**Review by frenck — DISMISSED** [review]
Thanks, @gryzli133 👍

File: source/_integrations/cover.mysensors.markdown [hunk 3]
```diff
@@ -149,3 +149,9 @@ void receive(const MyMessage &message) {
   }
 }
 ```
+
+## Sketch example with position measurement based on motor running time
+
+This sketch is ideally for star topology wiring. You can run up to 12 covers with a single Arduino Mega board and some relays. All you need to set is one line of parameters for one Cover. However, you can also use it for a single cover based on an Arduino Nano or even an ESP8266 board.
+
+[Check out the code on github.](https://github.com/gryzli133/RollerShutterSplit)
```
Comment (frenck) [inline source/_integrations/cover.mysensors.markdown]:
```suggestion
[Check out the code on Github.](https://github.com/gryzli133/RollerShutterSplit)
```

File: source/_integrations/cover.mysensors.markdown [hunk 4]
```diff
@@ -149,3 +149,9 @@ void receive(const MyMessage &message) {
   }
 }
 ```
+
+## Sketch example with position measurement based on motor running time
+
+This sketch is ideally for star topology wiring. You can run up to 12 covers with a single Arduino Mega board and some relays. All you need to set is one line of parameters for one Cover. However, you can also use it for a single cover based on an Arduino Nano or even an ESP8266 board.
+
+[Check out the code on Github.](https://github.com/gryzli133/RollerShutterSplit)
```
Comment (frenck) [inline source/_integrations/cover.mysensors.markdown]:
```suggestion
[Check out the code on GitHub.](https://github.com/gryzli133/RollerShutterSplit)
```

Comment (frenck) [inline source/_integrations/cover.mysensors.markdown]:
```suggestion
[Check out the code on GitHub.](https://github.com/gryzli133/RollerShutterSplit)
```

**Review by frenck — APPROVED** [review]

**Pull request merged**